src/Service.Tests/CosmosTests/QueryTests.cs

```diff
@@ -307,11 +306,11 @@ public async Task GetByPrimaryKeyWhenEntityNameDoesntMatchGraphQLType()
             string id = _idList[0];
             string query = @$"
 query {{
-    star_by_pk (id: ""{id}"", _partitionKeyValue: ""{id}"") {{
+    planet_by_pk (id: ""{id}"", _partitionKeyValue: ""{id}"") {{
```

Comment:
test method comments are now out of date. Did `star_by_pk` not work here? why can't we keep it how it was?

Reply:
it gives you wrong impression that, you can query entity directly which is not true, in case of CosmosDB. In case of CosmosDB, you can query only at container level entity.